src/stories/dropdown.js

```diff
@@ -27,20 +27,40 @@ storiesOf('Dropdown', module)
     `,
     () => (
       <div className="uk-margin-bottom">
-        <Dropdown>
+        <Dropdown component="div">
```

Comment:
since the default value is 'div' the prop `component` can be removed from this example